### PR TITLE
July 2019 news

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-team Please add your June work.

(refs #11446)

### CODE REVIEW

**Comment by aeslaughter** [comment]
@idaholab/moose-team Did people see this, please add something if you have done something.

**Comment by permcody** [comment]
I will add stuff tomorrow (Note: I'm not Derek so I actually will).

**Comment by permcody** [comment]
@aeslaughter - I probably broke the build - but I want to reference the new "detail" stuff for people pushing to get SQA done this month. I noticed that that documentation is still missing on the documenting MOOSE page.

**Comment by aeslaughter** [comment]
It is there now: https://mooseframework.inl.gov/python/MooseDocs/generate.html

File: modules/doc/content/newsletter/2019_07.md [hunk 1]
```diff
@@ -42,3 +73,50 @@ Two simple generic constant materials were also added that can be used in array
...
+
+Makefile rules have been improved so that additionally parallelism is possible when using systems with newer
+GNU Make installed. Users on many flavors of Linux may notice significant reductions in build times when building
+applications that use several libraries or modules in MOOSE.
+
+## SQA Push
+
+The MOOSE team is making a big push to get all tests documented in the moose/test folder. This activity started
+in June and will continue through July. As part of this effort, we have extended/improved the syntax available
+for documenting tests. If you are working on SQA, please see the new syntax available [here](https://mooseframework.inl.gov/python/MooseDocs/generate.html).
```
Comment (aeslaughter) [inline modules/doc/content/newsletter/2019_07.md]:
```suggestion
for documenting tests. If you are working on SQA, please see the new syntax available in [MooseDocs/generate.md].
```

Comment (aeslaughter) [reply modules/doc/content/newsletter/2019_07.md]: @permcody This is how you should reference the content.

**Comment by YaqiWang** [comment]
There is a typo in array variable section due to a recent change: *component* should be *components*. Maybe @permcody can update it when addressing the comments.

File: modules/doc/content/newsletter/2019_07.md (hunk 1 above)
Comment (permcody) [reply modules/doc/content/newsletter/2019_07.md]: The content still hasn't been updated yet. Do you have a PR coming for that?

Comment (aeslaughter) [reply modules/doc/content/newsletter/2019_07.md]: I made a suggestion for you to look at, is it not there?

**Review by permcody — APPROVED** [review]

**Pull request merged**